pmm-app/src/pmm-add-instance-app-panel/DiscoveryPanel/InstancesTable.tsx

```diff
@@ -0,0 +1,80 @@
+import { Table } from 'antd';
+import React from 'react';
+
+const getColumnType = type => {
+  switch (type) {
+    case 'DISCOVER_RDS_MYSQL':
+      return 'MySQL';
+    case 'DISCOVER_RDS_POSTGRES':
+      return 'Postgres';
+    case 'DISCOVER_RDS_INVALID':
+      return 'Unknown type';
+    default:
+      return 'Unknown type';
+  }
+};
+
+const InstancesTable = props => {
+  const { instances, selectInstance, credentials } = props;
```

Comment:
let's rename `selectInstance` to `onInstanceSelect` since it's a function to handle event.

Reply:
and use this convention for other event handlers